truss/cli/cli.py

```diff
@@ -464,6 +470,12 @@ def _extract_request_data(data: Optional[str], file: Optional[Path]):
     required=False,
     help="Name of the remote in .trussrc to push to",
 )
+@click.option(
```

Comment:
we don't need these extra CLI commands. The base domain will automatically get pulled out from remote.